geaflow/geaflow-dsl/geaflow-dsl-runtime/src/test/resources/expect/gql_algorithm_008.txt

```diff
@@ -0,0 +1,4 @@
+1,0
```

Comment:
It is best to produce at least one result, so that the unit test is more comprehensive.